src/blobs.js

```diff
@@ -1867,6 +1867,7 @@ div#${prntid} div.dialogs_container {
 div.toolbox_inner_cls {
    height: calc(100% - 38px);
    overflow-y: scroll;
+   overflow-x: hidden;
```

Comment:
This is to prevent horizontal scrollbar for the whole toolbox, but a horizontal scrollbar for just a `submit-button-row` is allowed if the buttons get big enough, right? Just checking

Reply:
exactly. the toolbox has a fixed width anyways so i dont think its scrollbar served any real purpose